src/webgl/context.js

```diff
@@ -195,18 +195,24 @@ export function glContextWithState(gl, {scissorTest, framebuffer}, func) {
  * @param {WebGLRenderingContext} gl - context
  * @return {Object} - 'vendor' and 'renderer' string fields.
  */
+let lastGLContext = null;
```

Comment:
Nit: Technically the JSDoc contextually belong to the next definition `lastGLContext`. We are not generating docs so maybe not a big deal.